src/server.ts

```diff
@@ -12,6 +13,7 @@ export async function initServer() {
   const server = new ApolloServer({
     typeDefs,
     resolvers,
+    formatError: formatError,
```

Comment:
```suggestion
    formatError,
```
Quando a variável tem o mesmo nome do parametro podemos simplificar!